provider/cldr/src/transform/time_zones/mod.rs

```diff
@@ -37,7 +37,7 @@ impl TryFrom<&dyn CldrPaths> for TimeZonesProvider {
     fn try_from(cldr_paths: &dyn CldrPaths) -> Result<Self, Self::Error> {
         let mut data = vec![];
 
-        let path = cldr_paths.cldr_dates()?.join("main");
+        let path = cldr_paths.cldr_dates("gregorian")?.join("main");
```

Comment:
Question: Is there time zone data in other calendars, or is it always stored in the gregorian calendar data bundle?

Reply:
Not that I know of: https://github.com/unicode-cldr/cldr-cal-buddhist-full/tree/master/main/af

Reply:
(I linked to the wrong repo, but the point still stands when looking at the cldr-json repo)